extensions/smallrye-metrics/runtime/src/main/java/io/quarkus/smallrye/metrics/runtime/SmallRyeMetricsServlet.java

```diff
@@ -18,18 +20,23 @@
     @Inject
     MetricsRequestHandler metricsHandler;
 
+    @ConfigProperty(name = "quarkus.servlet.context-path", defaultValue = "/")
+    String appContextPath;
+
     @Override
     protected void doOptions(HttpServletRequest req, HttpServletResponse resp) throws IOException {
         doGet(req, resp);
     }
 
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String metricsPath = getInitParameter("metrics.path");
+        String contextPath = (appContextPath.equals("/") ? "" : appContextPath) + metricsPath;
```

Comment:
I think we need to be a bit more careful about the `appContextPath` ending with a `/` or not?

Reply:
Ok I guess just stripping off a trailing `/` from `appContextPath` will  do the work

Reply:
Done